src/App/hooks/useFetchPoolStats.ts

```diff
@@ -0,0 +1,297 @@
+import { useEffect, useState, useContext } from 'react';
```

Comment:
This seems like its copied from `PoolCard` which is fine, but lets make sure that we remove this from PoolCard.tsx and use this implementation there as well.

Reply:
Will do once the review is completed. The original code in PoolCard.tsx was written by @benwolski